pegasus/test/test_script_constants.rb

```diff
@@ -36,4 +36,12 @@ def test_hoc?
     assert ScriptConstants.hoc?(ScriptConstants::ICEAGE_NAME)
     assert !ScriptConstants.hoc?(ScriptConstants::COURSE4_NAME)
   end
+
+  def test_category
+    assert_equal ScriptConstants.categories(ScriptConstants::HOC_NAME), ['hoc']
+    assert_equal ScriptConstants.categories(ScriptConstants::STARWARS_NAME), ['hoc']
+    assert_equal ScriptConstants.categories(ScriptConstants::FLAPPY_NAME), ['hoc', 'flappy']
+    assert_equal ScriptConstants.categories(ScriptConstants::COURSE1_NAME), ['csf']
+    assert_equal ScriptConstants.categories(ScriptConstants::CSP_UNIT1_NAME), ['csp']
```

Comment:
The pattern is `assert_equal expected, actual`, not `assert_equal actual, expected`.
http://ruby-doc.org/stdlib-2.1.0/libdoc/test/unit/rdoc/Test/Unit/Assertions.html#method-i-assert_equal